modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/index.tsx

```diff
@@ -66,7 +69,23 @@ export default function Action({
 
 		const {
 			title = Liferay.Language.get('an-error-occurred'),
-		} = (await response.json()) as {title?: string};
+			detail,
+		} = (await response.json()) as {detail: string; title?: string};
```

Comment:
Would be easier to understand the code if we set the constant type instead of  use `as`:
```suggestion
		}: {detail: string; title: string} = await response.json();
```